internal/pkg/cli/env_init.go

```diff
@@ -69,11 +71,18 @@ func (opts *InitEnvOpts) Execute() error {
 		return err
 	}
 
+	// TODO: move to opts method?
+	identity, err := opts.identity.Get()
+	if err != nil {
+		return fmt.Errorf("failed to get identity: %w", err)
```

Comment:
nit: One thing I've been trying to change as I'm touching old code is getting rid of the "failed to" prefix from our errors
> When adding context to returned errors, keep the context succinct by avoiding phrases like "failed to", which state the obvious and pile up as the error percolates up through the stack:

https://github.com/uber-go/guide/blob/master/style.md#error-wrapping